### PR TITLE
 docs(Content): Remove content that is now displayed in storybook

### PR DESCRIPTION
**Pull request by markhigham**
## Overview

Remove any content that is now hosted in storybook. In fact basically redirect everything to storybook

## Reason
We want to reduce the overhead of maintaining this static site and use Storybook to showcase components _and_ provide guidance and examples.

## Work carried out
- [x] Remove content and links from the home page
- [ ] Remove all content. This PR doesn't remove any deep links.

## Screenshot

localhost_3000_(Macbook)

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -316,125 +64,13 @@ export const Home: React.FC<HomeProps> = ({
...
-          {section5Buttons.map(({ href, title }) => {
-            return (
-              <Button key={title} href={href}>
-                {title}
-              </Button>
-            )
-          })}
-        </StyledButtonGroup>
+
+      <Section title="301 Moved Permanently">
```
Comment (m7kvqbe1) [inline pages/index.tsx:70]: Can we change this to be more succinct:

Screenshot 2024-04-29 at 08 55 24

File: pages/index.tsx [hunk 2]
```diff
@@ -286,15 +34,15 @@ export const Home: React.FC<HomeProps> = ({
       and its source code is available on GitHub."
       externalLinks={[
         <FooterExternalLink
-          icon={<GitHub />}
```
Comment (m7kvqbe1) [inline pages/index.tsx:289]: Do you have Prettier setup? Seems strange that it would have changed this?

File: pages/index.tsx [hunk 3]
```diff
@@ -1,282 +1,30 @@
...
-import { Badge, BADGE_VARIANT } from '../components/presenters/Docs/Badge'
-import { LayoutHomepage } from '../components/layouts/Homepage'
-import { contentful } from '../services/contentful'
-import HOMEPAGE_BY_ID_QUERY from '../graphql/queries/HomepageByID.graphql'
-import NAVIGATION_BY_ID_QUERY from '../graphql/queries/NavigationByID.graphql'
-import {
-  Navigation as NavigationType,
-  Homepage as HomepageType,
-} from '../graphql'
+import {Footer} from '../components/presenters/Docs/Footer'
```
Comment (m7kvqbe1) [inline pages/index.tsx:4]: There should be white space here either side of the brackets - is Prettier doing something weird?

**Review by m7kvqbe1 — COMMENTED** [review]
Looks good - just a comment about the content and some weird auto formatting.

File: pages/index.tsx (hunk 3 above)
Comment (markhigham) [reply pages/index.tsx:4]: Yes. This must've been Webstorm's default formatting

**Review by m7kvqbe1 — APPROVED** [review]

**Pull request merged**